src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <p className="Person__age">
+      {person.age && (
+        `I am ${person.age}`
+      )}
+    </p>
+
+    {!person.isMarried ? (
+      <p className="Person__partner">
+        I am not married
+      </p>
+    ) : (
+      <p className="Person__partner">
+        {person.sex === 'm'
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`}
```

Comment:
You can take out wife/husband and put it into variable
```
const status = person.sex === 'm' ? 'husband' : 'wife';

<>
  {`${person.partnerName} is my ${status}`}
</>
```

Other more advanced way: 
```
const SexStatuses = { 'm': 'husband', 'f': 'wife' };

<>
  {`${person.partnerName} is my ${SexStatuses[person.sex]}`}
</>
```